pkg/scheduler/core/generic_scheduler.go

```diff
@@ -520,8 +516,8 @@ func (g *genericScheduler) findNodesThatFit(ctx context.Context, state *framewor
 		workqueue.ParallelizeUntil(ctx, 16, int(allNodes), checkNode)
 
 		filtered = filtered[:filteredLen]
-		if len(errs) > 0 {
-			return []*v1.Node{}, FailedPredicateMap{}, framework.NodeToStatusMap{}, errors.CreateAggregateFromMessageCountMap(errs)
+		if err := errCh.ReceiveError(); err != nil {
```

Comment:
This doesn't look right, we used to return all predicates errors to the users, but here it changes to a single error.

Reply:
Ah I misread, `err` is for unexpected internal error, not PredicateFailure. Nevermind then.